src/components/Comments/WriteComment.js

```diff
@@ -28,7 +28,10 @@ const WriteComment = ({
   inReplyCommentView = false,
   showProfileConnector = true,
   loadingPost = false,
-  postId = ''
+  postId = '',
+  isKeyboardOpen = false,
+  isViewOnly = false,
+  withAnonymityLabel = true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new props `isKeyboardOpen`, `isViewOnly`, and `withAnonymityLabel` have been added to the `WriteComment` component. Ensure that these props are being passed correctly from the parent components and that they have appropriate default values if not provided.



<!-- This is an auto-generated comment by OSS CodeRabbit -->